### ISSUE TITLE
Better error message for conduit dashboard command when kubectl proxy is already running

### ISSUE THREAD

**Issue by dadjeibaah** [runconduit/conduit#220]
When you have a `kubectl proxy` running in another tab in the terminal and try to run `conduit dashboard` again. You receive this error
```bash
conduit git:(master) bin/conduit dashboard
Running `kubectl proxy -p 8001`

FATA[0000] Failed to start kubectl proxy: error waiting for kubectl to start the proxy. kubectl returned [], error: reached end of stream while waiting for character [
] in output [] of command: EOF
```

Suggestion by @olix0r is to no longer shell out `kubectl proxy` command when running `conduit dashboard`

**Comment by briansmith** [comment]
A very simple workaround to start with: Change the default port number from 8001 to any other value. `kubectl proxy` also picks 8001 as its default port and if both things use the same default then there will be a lot of conflicts. Conversely, if each thing has a different default then users will experience fewer conflicts.

**Comment by wmorgan** [comment]
We should improve the error message regardless

**Comment by briansmith** [comment]
I filed #291 about using a different port number to avoid this problem more often. I agree with @wmorgan that we should fix the error message, however IMO #291 is more important because it's better to avoid the problem in the first place.

**Comment by klingerf** [comment]
The error message for this type of failure now looks like this:

```
$ conduit dashboard -p 8001
Failed to initialize proxy: Failed to listen with proxy: Failed to listen via proxy server: listen tcp 127.0.0.1:8001: bind: address already in use
```

Given that we run the proxy on an ephemeral port by default, and that the error message for a port conflict is clearer (albeit somewhat verbose), I'm going to close this.
